custom_components/tuya_local/devices/miheat_th11wf_thermostat.yaml

```diff
@@ -130,6 +130,21 @@ entities:
       - id: 123
         type: boolean
         name: switch
+  - entity: sensor
+    translation_key: time_remaining
+    class: duration
+    category: diagnostic
+    dps:
+      - id: 119
+        type: integer
+        name: sensor
+        unit: min
+        mapping:
+          - dps_val: 0
+            constraint: Boost
```

Comment:
constraint needs to reference a dp on the same entity. This is referencing a different entity without specifying the dp name.

But is this really required? No other devices have this, they just show the data the device is sending.